src/components/elements/TimePicker.js

```diff
@@ -70,21 +69,24 @@ class TimeAndDatePicker extends Component {
           }}
                 />
         <FlatButton
-          label='submit'
-          primary
-          onClick={() => this.loadUncorrelatedEvents()}
+          label='Select time range'
```

Comment:
Can the user enter date and time manually, without the time pickers?

Reply:
it seems that material-ui does not support the typing in